package.json

```diff
@@ -2,6 +2,10 @@
   "devDependencies": {
     "markdown-link-check": "3.10.3",
     "markdown-toc": "^1.2.0",
-    "markdownlint-cli": "0.31.0"
+    "markdownlint-cli": "0.31.0",
+    "prettier": "^3.0.0"
+  },
+  "prettier": {
+    "proseWrap": "preserve"
```

Comment:
Note that eventually we can to change this option to `always`. That makes for a better experience viewing PR diffs in GitHub. For now, to minimize changes, we should keep `preserve`.